opdevnet/l1.go

```diff
@@ -58,6 +58,8 @@ func (cfg *L1Config) Run(ctx context.Context, env *environment.Env, logger log.L
 		func(_ *ethconfig.Config, nodeCfg *node.Config) error {
 			nodeCfg.WSHost = cfg.URL.Hostname()
 			nodeCfg.WSPort = int(cfg.URL.PortU16())
+			// TODO: make L1 http port configurable
+			nodeCfg.HTTPPort = 39473
```

Comment:
I opted to include this change in a follow-up PR for now, lmk if you think I should update it in this PR instead